proposals/presentation-message-format.md

```diff
@@ -0,0 +1,1330 @@
+# Addition of Presentation Request/Response to the W3C Verifiable Credentials specification
+- Authors:
+  - Daniel McGrogan [daniel.mcgrogan@workday.com](mailto:daniel.mcgrogan@workday.com)
+  - Jonathan Reynolds [jonathan.reynolds@workday.com](mailto:jonathan.reynolds@workday.com)
+- Last updated: 2019-12-18
+## Status
+- Status: **PROPOSAL**
+- Status Date: 2019-12-18
+- Status Note: Draft proposal for initial feedback
+
+## Abstract
+The [W3C Verifiable Credentials (hereafter VC) specification](https://www.w3.org/TR/vc-data-model) does not currently outline how credential data should be requested by a Verifier. This document outlines the approach taken at Workday and proposes it as an addition to the VC spec.
+
+This document deals only with format of Presentation messages. Presentation protocol should be decoupled from its format and, as such, is dealt with in a separate document.
+
+## Contents
+  * [Proposal](#proposal)
+    + [Introduction](#introduction)
+    + [Selective Disclosure](#selective-disclosure-via-individually-signed-attributes)
```

Comment:
> Selective Disclosure

Are we using these terms with the same meanings attributed to them here? https://w3c-ccg.github.io/data-minimization/

Reply:
Yes that is the intention. In Workday we sign attributes individually in a credential and request individual fields from a schema in the presentation.